test/throws.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+var expect  = require('chai').expect;
+var lost    = require('../lost');
+var postcss = require('postcss');
+var CssSyntaxError = require('postcss').CssSyntaxError;
+
+module.exports = function throws( input, message, opts ) {
+  var processor = postcss([lost(opts)]);
+  expect(function(){
+    return processor.process(input).css;
+  }).to.throw(CssSyntaxError, message);
+};
```

Comment:
@peterramsing not sure why but one of the travis builds failed to start for some reason. I wasn't sure how you restarted it and it doesn't appear to retry on its own. Any ideas?